confignode/src/main/java/org/apache/iotdb/confignode/manager/load/LoadManager.java

```diff
@@ -411,6 +420,257 @@ public List<TDataNodeConfiguration> getOnlineDataNodes(int dataNodeId) {
+    if(allConfigNodes == null){
+      return 0;
+    }
+    for (TConfigNodeLocation configNodeLocation : allConfigNodes) {
+      String name =
+              "EndPoint("
+                      + configNodeLocation.getInternalEndPoint().ip
+                      + ":"
+                      + configNodeLocation.getInternalEndPoint().port
+                      + ")";
+      MetricsService.getInstance()
+              .getMetricManager()
+              .getOrCreateGauge(
+                      Metric.CONFIG_NODE.toString(),
+                      MetricLevel.IMPORTANT,
+                      Tag.NAME.toString(),
+                      name,
+                      Tag.STATUS.toString(),
+                      "1:online,0:unkonwn")
+              .set(1);
+    }
+    return allConfigNodes.size();
+  }
+
+  public int getRunningDataNodesNum() {
+    List<TDataNodeConfiguration> allDataNodes = getOnlineDataNodes(-1);
+    if(allDataNodes == null){
+      return 0;
+    }
+    for (TDataNodeConfiguration dataNodeInfo : allDataNodes) {
+      TDataNodeLocation dataNodeLocation = dataNodeInfo.getLocation();
+      String name =
+              "EndPoint("
+                      + dataNodeLocation.getClientRpcEndPoint().ip
+                      + ":"
+                      + dataNodeLocation.getClientRpcEndPoint().port
+                      + ")";
+      MetricsService.getInstance()
+              .getMetricManager()
+              .getOrCreateGauge(
+                      Metric.DATA_NODE.toString(),
+                      MetricLevel.IMPORTANT,
+                      Tag.NAME.toString(),
+                      name,
+                      Tag.STATUS.toString(),
+                      "1:online,0:unkonwn")
+              .set(1);
+    }
+    return allDataNodes.size();
+  }
+
+  public int getUnknownConfigNodesNum() {
+    List<TConfigNodeLocation> allConfigNodes = getUnknownConfigNodes();
+    if(allConfigNodes == null){
+      return 0;
+    }
+    for (TConfigNodeLocation configNodeLocation : allConfigNodes) {
+      String name =
+              "EndPoint("
+                      + configNodeLocation.getInternalEndPoint().ip
+                      + ":"
+                      + configNodeLocation.getInternalEndPoint().port
+                      + ")";
+      MetricsService.getInstance()
+              .getMetricManager()
+              .getOrCreateGauge(
+                      Metric.CONFIG_NODE.toString(),
+                      MetricLevel.IMPORTANT,
+                      Tag.NAME.toString(),
+                      name,
+                      Tag.STATUS.toString(),
+                      "1:online,0:unkonwn")
+              .set(0);
+    }
+    return allConfigNodes.size();
+  }
+
+  public int getUnknownDataNodesNum() {
+    List<TDataNodeConfiguration> allDataNodes = getUnknownDataNodes(-1);
+    if(allDataNodes == null){
+      return 0;
+    }
+    for (TDataNodeConfiguration dataNodeInfo : allDataNodes) {
```

Comment:
I think the confignode/datanode ip metric is too many. 
And I do not think the online time is important.  
In current implement, if I delete some datanode, but the metric exists.